.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Run lint
+        run: pnpm run lint
+
+      - name: Run type-check
+        run: pnpm run type-check
+
+      - name: Build application
+        run: pnpm run build
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)